src/main/resources/schemas/schema-form.json

```diff
@@ -23,9 +23,9 @@
       "description": "'SOAPAction' HTTP header send when invoking WS",
       "type": "string"
     },
-    "charset": {
-      "title": "Charset",
-      "description": "This charset will be appended to the Content-Type header value",
+    "contentType": {
+      "title": "Content-Type",
+      "description": "Overwrite the default Content-Type header value (default value = text/xml)",
```

Comment:
You can set the default value in the schema:

"default":  "text/xml"

Reply:
I wasn't aware of that. I will change the implementation.
I have a question releated to migrating from the current version (1.5.0 with charset option) to the new version (1.6.0-SNAPSHOT without charset option). When I replace the policy with the new version I get an error in the management UI. Is this caused by the backend store value for the charset option? If yes, how can I correct this and do I need to automate this migration for the next release?

Reply:
I think you can still have a charset field within `SoapTransformerPolicyConfiguration` to map the old value to the bean, no?
If not, you will have to manage the serialization / deserialization by yourself (or exclude charset field using a jackson annotation on the `SoapTransformerPolicyConfiguration` bean).

Reply:
@brasseld 
I did some extra research on the topic and I think we can go for a list and keep the charset field; as suggested by you.

SOAP version 1.1 uses content-type "text/xml" and default charset "us-ascii"
SOAP version 1.2 uses content-type "application/soap+xml" and default charset "utf-8"

The content-type "Multipart-Related" is also supported with the type "application/xop+xml" but I believe that this will be difficult to implement and have limited use cases.

So I suggest to have a list with the values:
- text/xml
- application/soap+xml

Do you agree?

Reply:
I would say that we can start with these content-types and if one is missing, someone will ask us to add it!

So yes, agreed!

Reply:
Top!
Please review the latest commit a4c9f40 which contains the agreed implementation.